scilpy/image/datasets.py

```diff
@@ -87,71 +82,85 @@ def is_voxel_in_bound(self, i, j, k):
                 0 <= j < (self.dim[1]) and
                 0 <= k < (self.dim[2]))
 
-    def voxmm_to_idx(self, x, y, z, origin):
+    def _idx_to_closest_coord(self, i, j, k):
```

Comment:
Not sure about the name, gives the impression there is a rounding operation or something... Maybe `_clip_idx_to_bounds` ?

Reply:
Well there is a rounding operation; a floor option. (In vox_to_idx). How could the doc be clearer?

Reply:
Ok, yes, I like it!